components/director/pkg/graphql/schema.graphql

```diff
@@ -783,6 +895,10 @@ type Mutation {
 	"""
 	addAPIDefinition(applicationID: ID!, in: APIDefinitionInput! @validate): APIDefinition! @hasScopes(path: "graphql.mutation.addAPIDefinition")
 	"""
+	Temporary name before doing breaking change
```

Comment:
mark that `addAPIDefinition` will be changed and at the end we will have only `addAPIDefinitionToPackage`.
The same for `addEventDefinition`